src/components/HomePage/HomepageTestimonials.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="testimonials">
+    <div layout="column u4">
+      <div v-for="t in testimonials" :key="t.name" layout="row u1">
```

Comment:
```suggestion
    <div layout="row u4">
      <div v-for="t in testimonials" :key="t.name" layout="column u1">
```

i believe you want the "row" and "column" swapped here, to have a "row of columns" instead of "column of rows" here.